modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach errors
+
+.Problem
```

Comment:
Although it's kind of neat laid out like this, I'm not sure whether this is acceptable presentation per a mod docs module type.

And unless the rest of our troubleshooting docs follow this Problem/Solution format (which I don't think it does), I'd probably avoid introducing it now.

Reply:
Yep, you're right. It wasn't something we have used elsewhere. I thought it might help with readability but def not necessary. Removed.